downstream/modules/platform/con-building-an-execution-environment-in-a-disconnected-environment.adoc

```diff
@@ -16,33 +16,11 @@ link:https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_pl
 
 * registry.redhat.io for access to the base container images.
 
-* Red Hat does not have an IP list for the container registry domains or cdn-ubi.redhat.com as for cdn.redhat.com. 
-To firewall off the registry and/or UBI repositories, use a proxy (e.g. squid) and allow-list the following domains:
-
-//* `registry.access.redhat.com`
-* `registry.redhat.io`
-//* `registry.connect.redhat.com`
-* `sso.redhat.com`
-* `cdn-ubi.redhat.com`
-* `cdn.quay.io`
-* `cdn01.quay.io`
-* `cdn02.quay.io`
-* `cdn03.quay.io`
-
-For additional information, see link:https://access.redhat.com/articles/1525183[Public CIDR Lists for Red Hat].
+Building an execution environment image in a disconnected environment requires mirroring content from these locations. See link:https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/2.4/html-single/red_hat_ansible_automation_platform_installation_guide/index#importing-collections-into-private-automation-hub_disconnected-installation[Importing Collections into private automation hub] for information on importing collections from Ansible Galaxy or automation hub into a private automation hub.
```

Comment:
The first line is missing an attribute. "Building an {ExecEnvShort} image in a disconnected environment..."

Also, a couple of missed attributes after the link: ..."for information on importing collections from {Galaxy} or {HubName} into a {PrivateHubName}."

Reply:
I have no idea how that happened. They were in the original copy. Nice catch! Thanks!